decidim/decidim
# Make it possible to allow some pages to display publicly even when organization access is limited

#### :tophat: What? Why?
Currently if you have configured the Decidim organization to "Force users to authenticate before access organization", some relevant pages become inaccessible for unauthenticated users.

An example of this is the is the "Find out more about cookies" link in the cookie notification (see the screenshots section). If the user cannot find out more about the cookies, how can you expect them to accept what is written there?

This PR implements a new checkbox for the static pages which allows administrators to configure specific pages to be accessible even without authentication. This configuration option is automatically applied to the `terms-and-conditions` page where the link in the cookie notification points to.

We also got a user feedback suggesting that this may be against GDPR that they cannot read the terms before accepting them.

#### Testing
- Create a new Decidim instance with the default seed content
- Go to the system administrator panel at `/system` and enable the "Force users to authenticate before access organization" configuration option for the organization
- Log out from the system administrator panel
- Go to the front page of the installation
- Click the "Find out more about cookies" link in the cookie notification (clear cookies if you don't see it)
- See that you cannot access the page where you should be able to "find out more about cookies".

#### :clipboard: Checklist

### :camera: Screenshots
An inaccessible link in the cookie notification before signing in:
![Cookie notification with a link](https://user-images.githubusercontent.com/864340/100548745-28af3480-3277-11eb-9043-4396d387be1d.png)
